reconcile/gql_definitions/terraform_cloudflare_dns/terraform_cloudflare_zones.gql

```diff
@@ -0,0 +1,64 @@
+# qenerate: plugin=pydantic_v1
+
+query CloudflareDnsZone {
+  zones: cloudflare_dns_zone_v1 {
+    identifier
+    zone
+    account {
+      name
+      type
+      description
+      providerVersion
+      enforceTwofactor
+      apiCredentials {
+        path
+        field
+        format
+        version
```

Comment:
You can use `... VaultSecret` fragment here to reduce the number of generated classes.